newIDE/app/src/AssetStore/InstallAsset.js

```diff
@@ -419,6 +419,14 @@ export const addSerializedExtensionsToProject = (
       'unserializeFrom',
       project
     );
+
+    if (serializedExtension.url) {
```

Comment:
Are you sure this can even happen? I'm not sure because the SerializedExtension type is a representation of this:

https://resources.gdevelop-app.com/extensions/DoubleClick.json

which does not contain the URL, as this is only in the "header":

https://resources.gdevelop-app.com/extensions/DoubleClick-header.json

The type was sadly misguiding you :(

Reply:
Indeed, I may have confused a few things when testing